### PR TITLE
Generate the update ID/alias upon creation

### PR DESCRIPTION
**Pull request by lmacken**
As opposed to when it's initially pushed to testing, which is how it is currently done.

### CODE REVIEW

File: bodhi/tests/functional/test_updates.py [hunk 1]
```diff
@@ -1147,7 +1147,7 @@ def test_edit_update(self, publish, *args):
         self.assertEquals(up['date_approved'], None)
         self.assertEquals(up['date_pushed'], None)
         self.assertEquals(up['locked'], False)
-        self.assertEquals(up['alias'], None)
+        self.assertEquals(up['alias'], u'FEDORA-2015-0002')
```
Comment (ralphbean) [inline bodhi/tests/functional/test_updates.py]: Do you want to use ``time.localtime`` here for the year?  I.e., will these tests break when we get to 2016?

**Comment by ralphbean** [comment]
Looks good all-in-all.  :+1:

File: bodhi/tests/functional/test_updates.py (hunk 1 above)
Comment (ralphbean) [inline bodhi/tests/functional/test_updates.py]: Or, use ``freezegun.freeze_time('2015-06-03')`` on the test.

Comment (lmacken) [inline bodhi/tests/functional/test_updates.py]: Yep, good call!

**Comment by lmacken** [comment]
Thanks for the review!

**Pull request merged**